build.sh

```diff
@@ -0,0 +1,3 @@
+sudo make libinstall
+gcc -shared -fPIC -o libflashrom.so jedec.o stm50.o w39.o w29ee011.o sst28sf040.o 82802ab.o sst49lfxxxc.o sst_fwhub.o edi.o flashchips.o spi.o spi25.o spi25_statusreg.o opaque.o sfdp.o en29lv640b.o at45db.o processor_enable.o chipset_enable.o board_enable.o cbtable.o internal.o it87spi.o it85spi.o sb600spi.o amd_imc.o wbsio_spi.o mcp6x_spi.o ichspi.o dmi.o serprog.o rayer_spi.o pony_spi.o bitbang_spi.o nic3com.o gfxnvidia.o satasii.o atavia.o it8212.o ft2232_spi.o usbblaster_spi.o pickit2_spi.o dummyflasher.o drkaiser.o nicrealtek.o nicintel.o nicintel_spi.o nicintel_eeprom.o ogp_spi.o buspirate_spi.o dediprog.o developerbox_spi.o satamv.o linux_mtd.o linux_spi.o ch341a_spi.o digilent_spi.o pcidev.o physmap.o hwaccess.o usbdev.o libflashrom.o layout.o flashrom.o udelay.o programmer.o helpers.o ich_descriptors.o fmap.o serial.o custom_baud.o print.o cli_common.o -lflashrom -lusb-1.0 -lusb -lpci
```

Comment:
This can be later implemented as a makefile rule. All these object files are probably kept in one variable `LIBFLASHROM_OBJS`, so it will look very nice when used. Please note that for future improvement @artur-rs